From 3757c6a3130c24b9cf10854b74bd9d3e2a91e392 Mon Sep 17 00:00:00 2001
From: Sean Cross <xobs@kosagi.com>
Date: Fri, 10 Oct 2014 16:32:11 +0800
Subject: [PATCH 21/39] scripts: Add custom destdir parameter

We don't want packages installed to /boot since it's FAT-formatted.
Instead, our diversion script as part of U-Boot will take care of that.
As a workaround, add the KBUILD_DESTDIR variable so that users can specify
the path to place the finished kernel.

Signed-off-by: Sean Cross <xobs@kosagi.com>
---
 scripts/package/builddeb | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/scripts/package/builddeb b/scripts/package/builddeb
index 4f113835f8fd..21918f3cea04 100755
--- a/scripts/package/builddeb
+++ b/scripts/package/builddeb
@@ -149,6 +149,12 @@ if [ ! -z $KBUILD_DESTDIR ]; then
 	mkdir -p "${tmpdir}/${KBUILD_DESTDIR}"
 fi
 
+if [ ! -z $KBUILD_DESTDIR ]; then
+	installed_image_path="${KBUILD_DESTDIR}/vmlinuz-$version"
+	installed_dtb_path="${KBUILD_DESTDIR}/vmlinuz-$version.dtb"
+	mkdir -p "${tmpdir}/${KBUILD_DESTDIR}"
+fi
+
 # Build and install the kernel
 if [ "$ARCH" = "um" ] ; then
 	mkdir -p "$tmpdir/usr/lib/uml/modules/$version" "$tmpdir/usr/bin" "$tmpdir/usr/share/doc/$packagename"
-- 
2.14.1

